x-pack/plugins/session_view/public/components/detail_panel_metadata_tab/index.tsx

```diff
@@ -433,6 +433,21 @@ export const DetailPanelMetadataTab = ({
                   </DetailPanelCopy>
                 ),
               },
+              {
+                title: orchestratorData.resource.parent.type && (
```

Comment:
be cautious when using `&&` on JSX, the title will output `0` when evaluating to `false`. Also looking at other examples looks like you don't even need to check for the title

Reply:
woops i forgot to remove when addressing Jack's comment above